swagger.yaml

```diff
@@ -7013,8 +7035,55 @@ definitions:
         description: Indication that the version is the recommended one.
       support_level:
         type: string
-        enum: [beta, production, maintenance]
+        enum: [beta, production, maintenance, end-of-life]
         description: Level of support of the version.
+  
+  release-source:
+    type: object
+    required:
+      - openshift_version
+      - multi_cpu_architectures
+      - upgrade_channels
+    properties:
+      openshift_version:
+        type: string
+        description: Version of the OpenShift cluster.
+        example: "4.14"
+      multi_cpu_architectures:
+        type: array
+        items:
+          type: string
+          enum: ['x86_64', 'aarch64', 'arm64','ppc64le','s390x','multi']
```

Comment:
I think t his should only include one of `aarch64` or `arm64` and probably shouldn't include `multi`, right?

Reply:
About `aarch64/arm64`, currently we have these archs available for release images - https://github.com/openshift/assisted-service/blob/59f99694cfb82f33f8e7461d02eb46323e5abbc4/swagger.yaml#L6996
Not sure about whether we should have both of them or only one